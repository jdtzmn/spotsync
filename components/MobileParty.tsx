import React, { Component } from 'react'
import MediaControl from './MediaControl'
import css from '../styles/components/mobileParty.scss'

class MobileParty extends Component {
  render() {
    return (
      <div>
        <div className={css.status}>
          <div className={css.container}>
            {/* top row */}
            <div className={css.topRow}>
              <div>
                #000000
              </div>
              <div className={css.position}>
                3:07
              </div>
            </div>
            {/* song status */}
            <div className={css.songStatus}>
              <div className={css.verticalAlign}>
                <div className={css.songImage} />
              </div>
              <div className={css.songDetails}>
                <h1 className={css.songName}>Believer</h1>
                <sub className={css.artistName}>Imagine Dragons</sub>
              </div>
              <MediaControl
                className={css.mediaControl}
                size='40px'
                color='#515151'
              />
            </div>
          </div>
        </div>
      </div>
    )
  }
}

export default MobileParty
