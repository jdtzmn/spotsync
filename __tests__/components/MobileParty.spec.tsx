import * as React from 'react'
import { shallow } from 'enzyme'
import MobileParty from 'components/MobileParty'

describe('MobileParty', () => {
  let wrapper
  beforeEach(() => {
    wrapper = shallow(<MobileParty />)
  })

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
