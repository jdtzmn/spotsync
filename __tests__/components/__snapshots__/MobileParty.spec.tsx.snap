// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MobileParty should render correctly 1`] = `
<div>
  <div>
    <div>
      <div>
        <div>
          #000000
        </div>
        <div>
          3:07
        </div>
      </div>
      <div>
        <div>
          <div />
        </div>
        <div>
          <h1>
            Believer
          </h1>
          <sub>
            Imagine Dragons
          </sub>
        </div>
        <MediaControl
          color="#515151"
          size="40px"
        />
      </div>
    </div>
  </div>
</div>
`;
